server/src/server.cpp

```diff
@@ -0,0 +1,64 @@
+#include <ros/ros.h>
+#include <client/time.h>
+#include <client/show.h>
+
+#include <vector>
+#include <iostream>
+
+client::show srv;
+//ros::NodeHandle *n_pointer = NULL;
+std::vector<ros::Subscriber> server_subscribers;
+ros::ServiceServer service_server;
+
+void time_callback(const client::time::ConstPtr& time)
+{
+    // 将接收到的消息打印出来
+    ROS_INFO("Client_name:%s   time:%lu ", time->name.c_str(), time->sec);
+}
+
+// service回调函数，输入参数request，输出参数response
+bool show_callback(client::show::Request &request, client::show::Response &response)
+{
+    if(request.request == 1) 

```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]